komsic/kowope
# #164980385 Initialize the ui part of the project

#### What does this PR do?
Initialize the ui project

#### Description of Task to be completed?
Add the ui folder to the project

#### How should this be manually tested?
Navigate to the [project](https://github.com/komsic/kowope/tree/ch-initialize-ui-project-164980385) and see whether the **ui** folder has been created

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#164980385](https://www.pivotaltracker.com/story/show/164980385/comments/201469210)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
